<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>About VolumanQ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="volumanq.qrc">
    <normaloff>:/resources/star_128x128.png</normaloff>:/resources/star_128x128.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-1</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
  background-color: white;
  border: 1px solid gray;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="appName">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>3</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>21</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>VolumanQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="appVersion">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>34</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0.1beta</string>
    </property>
   </widget>
   <widget class="QLabel" name="appLogo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>19</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="appDescription">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Simple volume indicator for Linux written on Qt 4 using ALSA. It is really useful for small desktop environments and window managers such as: LXDE, Fluxbox, PekWM and others.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;It is free and with open sources licensed under the MIT License.&lt;/p&gt;&lt;p&gt;2013 (c) Dmytro Dmytrenko, &lt;a href=&quot;mailto:dmytrenko.d@gmail.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;dmytrenko.d@gmail.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>300</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="icon">
    <iconset resource="volumanq.qrc">
     <normaloff>:/resources/help_128x128.png</normaloff>:/resources/help_128x128.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="volumanq.qrc"/>
 </resources>
 <connections/>
</ui>
